pyo3-ffi/src/descrobject.rs

```diff
@@ -68,6 +68,11 @@ extern "C" {
     pub fn PyWrapper_New(arg1: *mut PyObject, arg2: *mut PyObject) -> *mut PyObject;
 }
 
+#[inline]
+pub unsafe fn PyDictProxy_Check(op: *mut PyObject) -> c_int {
+    PyObject_TypeCheck(op, std::ptr::addr_of_mut!(PyDictProxy_Type))
+}
```

Comment:
This function doesn't exist in https://github.com/python/cpython/blob/3.13/Include/descrobject.h (indeed I couldn't find it anywhere). So we shouldn't add this.

```suggestion
```

Reply:
Removed!